run/helm/snap/README.md

```diff
@@ -0,0 +1,246 @@
+
+...and copy the link for the latest plugin release.
+
+<img src="https://cloud.githubusercontent.com/assets/6523391/21221622/69a08e6c-c2be-11e6-916f-f7179332b435.png" width="70%">
+  
+Then you add this link to the `autoload` parameter of `values.yaml` file.
+
+```
+$ cat <<EOF >> new_values.yaml
+autoload:
+  - https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/6/snap-plugin-collector-docker_linux_x86_64
+  - https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64
+EOF
+```
+
+You need to do the same with file publisher plugin.
+
+#### Step 2. Configuration of plugin
+
+Docker collector reads values from files residing on the host, therefore those files need to be mounted inside a container in which Snap is running. Docker collector needs access to files on the host:
+- `/var/run/docker.sock`
+- `/proc`
+- `/usr/bin/docker`
+- `/var/lib/docker`
+- `/sys/fs/cgroup`
+
+Usually you would add `volumeMounts` and `volumes` parameters to the Kubernetes manifest, but this Snap chart handles volumes mounting for you. The only thing to change is `mounts` parameter of chart values file.
+
+```
+$ cat <<EOF >> new_values.yaml
+mounts:
+  /var/run/docker.sock: /var/run/docker.sock
+  /proc: /proc_host
+  /usr/bin/docker: /usr/local/bin/docker
+  /var/lib/docker: /var/lib/docker
+  /sys/fs/cgroup: /sys/fs/cgroup
+privileged: true
+EOF
+```
+
+Format of mounts parameter is:
+```
+mounts:
+ <host_path_a>: <container_path_a>
+ <host_path_b>: <container_path_b>
+```
+
+We will also need privileged mode to access those files residing on the host.
+
+### Preparing Snap Task 
+
+The last thing to prepare is task manifest that will specify what metrics should be collected. If you want to use an exemplary task from this repo you can specify task URL the same way we provided plugins URL, simply by adding third element to the `autoload` list in `values.yaml` file so that final `new_values.yaml` file looks like this:
+
+```
+mounts:
+  /var/run/docker.sock: /var/run/docker.sock
+  /proc: /proc_host
+  /usr/bin/docker: /usr/local/bin/docker
+  /var/lib/docker: /var/lib/docker
+  /sys/fs/cgroup: /sys/fs/cgroup
+privileged: true
+autoload:
+  - https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/6/snap-plugin-collector-docker_linux_x86_64
+  - https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64
+  - https://raw.githubusercontent.com/mkuculyma/snap-integration-kubernetes/helm/examples/tasks/docker-file.yaml
+```
+
+The other way is to create and `/opt/snap` directory in the nodes on which Snap pods will run and place there task manifests. Everything that resides in this dir will be automatically loaded at the start of the container. Note: You need to copy the `/opt/snap` directory to all of the cluster nodes on which you wish to run Snap.
```

Comment:
I'm not sure if this is a recommended operation model for kubernetes. Do we have examples where other applications advocate the same? Seems like putting everything into a PV would be more reasonable.